test/Serilog.Sinks.PeriodicBatching.PerformanceTests/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+﻿{
+    "shadowCopy": false
+}
```

Comment:
This is needed in order to run performance tests under `net452` otherwise `BenchmarkDotNet` fails to find output dll's in the shadow folder.